### PR TITLE
[Wake Lock article] Document new WakeLockSentinel.released

### PR DESCRIPTION
**Pull request by tomayac**
Changes proposed in this pull request:

- Document new `WakeLockSentinel.released`.
- Consistently use space (dash) in URL.

### CODE REVIEW

**Review by jpmedley — APPROVED** [review]

File: src/site/content/en/blog/wake-lock/index.md [hunk 1]
```diff
@@ -167,6 +167,12 @@ window.setTimeout(() => {
 }, 5000);
 ```
 
+The `WakeLockSentinel` object has a property called `released` that
+indicates whether a sentinel has already been released.
+Its value is initially `false`, and changes to `true` once a `"release"`
+event is dispatched. This property helps web developers know when a lock
+has been released so that they do not need to keep track of this manually.
+
```
Comment (jpmedley) [inline src/site/content/en/blog/wake-lock/index.md:175]:
```suggestion
This property is not yet universally available. As of this writing, it's 
supported in Chrome 87 or later and Chromium forks based on it. 
Check [MDN](https://developer.mozilla.org/en-US/docs/Web/API/WakeLockSentinel/released#Browser_compatibility)
before using it.
 
```

Comment (jpmedley) [reply src/site/content/en/blog/wake-lock/index.md:175]: I wanted to say, "or caniuse", but it's not there yet.

**Review by jpmedley — CHANGES_REQUESTED** [review]
This needs to cover that the new property isn't available until Chrome 87.

File: src/site/content/en/blog/wake-lock/index.md [hunk 2]
```diff
@@ -167,6 +167,12 @@ window.setTimeout(() => {
 }, 5000);
 ```
 
+The `WakeLockSentinel` object has a property called `released` that
+indicates whether a sentinel has already been released.
+Its value is initially `false`, and changes to `true` once a `"release"`
+event is dispatched. This property helps web developers know when a lock
+has been released so that they do not need to keep track of this manually.
+It is available as of Chrome&nbsp;87.
```
Comment (tomayac) [inline src/site/content/en/blog/wake-lock/index.md:175]:
```suggestion
It is available as of Chrome&nbsp;87.

```

**Review by jpmedley — APPROVED** [review]

**Review by jpmedley — APPROVED** [review]

**Comment by tomayac** [comment]
@kaycebasques, could you approve the redirect, please, and then merge? Joe has approved the content. Thanks!

**Comment by jpmedley** [comment]
@kaycebasques Apparently I don't have the ability to approve redirects.

**Review by robdodson — APPROVED** [review]
redirect change LGTM

**Pull request merged**